c2pa/claim-examples/claim_create.json

```diff
@@ -0,0 +1,98 @@
+{
+    "vendor": "starlinglab",
```

Comment:
`"vendor"` is not defined in Claims. To specify the vendor name, we can use the Manifest label to specify it.

For example, `starlinglab:urn:uuid:{uuid}` (refer to the [section 10.1.1](https://c2pa.org/public-draft/C2PA_Specification.html#_use_of_jumbf) for the detailed description)

Reply:
@Irosenthol I guess this lands in here right?
```
"root": "starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967",
```

Reply:
Correct, vendor is not in the spec on its own,  but it is required for claim_tool to create labels.